src/views/Bills/components/Menu/styles.ts

```diff
@@ -14,14 +14,18 @@ export const ControlContainer = styled(Flex)`
   z-index: 1;
   background-color: ${({ theme }) => theme.colors.white2};
   min-width: 300px;
-  max-width: 500px;
+  max-width: 100%;
```

Comment:
why do we have style changes, can you revert them in this file and the ones below?